### PR TITLE
infra: remove target from ovirt_storage_connection

### PR DESCRIPTION
**Pull request by mnecas**
The reason for this is that the storage domain connection can be created with multiple targets, the issue comes when there is another connection with a different target than the user has specified by the storage in the YAML file.
https://bugzilla.redhat.com/show_bug.cgi?id=1943221

### CODE REVIEW

**Review by mwperina — APPROVED** [review]
Looks good to me, but we need a final ack from storage team about it

**Comment by mwperina** [comment]
Merging to unblock releasing for oVirt 4.4.6. If the solution is not good, we will need to fix it later

File: roles/infra/roles/storages/tasks/main.yml
Comment (barpavel) [inline roles/infra/roles/storages/tasks/main.yml:80]: pamareters --> parameters
:)

Comment (mnecas) [reply roles/infra/roles/storages/tasks/main.yml:80]: Fixed in: https://github.com/oVirt/ovirt-ansible-collection/pull/282

**Pull request merged**
